HealthcareSystem/GraphicalEditor/MainWindow.xaml.cs

```diff
@@ -126,33 +127,84 @@ private void saveMap()
 
         private void Edit_Display_Information(object sender, RoutedEventArgs e)
         {
+            this.ObjectTypeChooserComboBox.SelectedItem = DisplayMapObject.MapObjectType.ObjectTypeFullName;
             EditMode = !EditMode;
         }
 
-        private void Change_Display_Information(object sender, RoutedEventArgs e)
-        {          
-            string selectedItem = (string)chooser.SelectedItem;
-            DisplayMapObject.MapObjectType.ObjectTypeFullName = selectedItem;
+        public void ChangeEditButtonVisibility() {
```

Comment:
Why does this method change the button visibility directly, when it's already bound to the EditMode property? It looks like it's a fix to not show the edit button when a road is selected, but I feel like using both approaches is odd, and might warrant a redesign in the future.

Reply:
i think you don't have last version of this branch